test/functional/apps/discover/_shared_links.js

```diff
@@ -89,8 +89,8 @@ export default function ({ getService, getPageObjects }) {
       });
 
       it('gets copied to clipboard', async function () {
-        const isCopiedToClipboard = await PageObjects.discover.clickCopyToClipboard();
-        expect(isCopiedToClipboard).to.eql(true);
+        // This method is self-confirming, so we don't need an assertion.
+        await PageObjects.discover.clickCopyToClipboard();
```

Comment:
same as above, maybe indicate the self-confirming nature of this method in its name.